# Message describing the state of the buttons
Header header
uint8 ChestButton
uint8 LFoorBumperLeft
uint8 LFootBumperRight
uint8 RFootBumperLeft
uint8 RFootBumperRight
uint8 HeadTouchFront
uint8 HeadTouchMiddle
uint8 HeadTouchRear